src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -379,5 +390,23 @@ export const locations = {
     france: `${paris.display_name}, ${paris.country}`,
     guernsey: `${guernsey.display_name}, ${guernsey.country}`,
 }
+export const continents = {
+    All: all_offices,
```

Comment:
```suggestion
    all: all_offices,
```

can we make this lowercase

Reply:
Sure @prince-deriv  I have updated this one.